src/views/2-Monitoring/MonitoringMobile.vue

```diff
@@ -69,13 +69,13 @@
         <h3 v-html="text.title4" />
       </div>
       <p v-html="text.paragraph12" />
-      <Chapter2Icons2 />
+      <Chapter2Icons3 />
       <p v-html="text.paragraph13" />
     </div>
     <div class="box">
       <div class="viz-title-wrapper">
         <p class="viz-title">
-          <span class="yellow">Daily temperature</span> of river reaches in 2019
+          <span class="yellow">Variability</span> of stream temperature in 2019
```

Comment:
Should it be "in" instead of "of"? "Variability in stream temperatures in 2019", but then that's a double "in"...suggesting "Stream temperature variability in 2019"